### PR TITLE
commit reset changes

### PR DESCRIPTION
**Pull request by auriium**

### CODE REVIEW

**Comment by auriium** [comment]
WHY IS IT FAILING

**Comment by A248** [comment]
The build is failing because the updateExperience, updateCurrentKillstreak, and updateHighestKillstreak methods do not exist, and also because the schema does not include experience, killstreak, or highest killstreak in kitpvp_statistics

**Comment by auriium** [comment]
Well, that's uhm.. obvious, to say the least? I'm really confused as to _why_ those methods do not exist

**Comment by auriium** [comment]
This even installs / packages correctly on my laptop doing a full install

File: mod-kitpvp/src/main/java/gg/solarmc/loader/kitpvp/OnlineKitPvp.java [hunk 1]
```diff
@@ -26,15 +26,20 @@
...
         this.kills = kills;
         this.deaths = deaths;
         this.assists = assists;
-
+        this.experience = experience;
+        this.currentKillstreaks = currentKillstreaks;
+        this.highestKillstreaks = highestKillstreaks;
     }
 
+    private volatile int experience;
```
Comment (A248) [inline mod-kitpvp/src/main/java/gg/solarmc/loader/kitpvp/OnlineKitPvp.java]: Fields should be above the constructor

**Review by A248 — APPROVED** [review]
Once you fix this and commit it you can merge

**Review by A248 — APPROVED** [review]

**Pull request merged**